tests/conftest.py

```diff
@@ -1,64 +1,89 @@
 import json
-import os
+import logging
 from contextlib import contextmanager
-from logging import info
 from pathlib import Path
 
 import pytest
 from plumbum import local
 from plumbum.cmd import docker
 
-DOCKER_IMAGE_NAME = os.environ.get("DOCKER_IMAGE_NAME", "docker-socket-proxy:local")
+_logger = logging.getLogger(__name__)
 
 
 def pytest_addoption(parser):
     """Allow prebuilding image for local testing."""
-    parser.addoption("--prebuild", action="store_const", const=True)
+    parser.addoption(
+        "--prebuild", action="store_true", help="Build local image before testing"
+    )
+    parser.addoption(
+        "--image",
+        action="store",
+        default="docker-socket-proxy-test:latest",
```

Comment:
It should be:

```suggestion
        default="test:docker-socket-proxy",
```

It feelos good IMHO having all test images under the same namespace.

Reply:
:thinking: ok, I will change that.